app/classifier/tasks/survey/index.cjsx

```diff
@@ -84,7 +87,13 @@ module.exports = React.createClass
   handleAnnotation: (choice, answers, e) ->
     filters = JSON.parse JSON.stringify @state.filters
     value = @props.annotation.value?.slice?(0) ? []
-    value.push {choice, answers, filters}
+    # @ is undefined within the scope of findIndex
+    currentSelection = @state.selectedChoiceID
+    annotationIndex = value.findIndex (value) -> value.choice is currentSelection
```

Comment:
`Array.find` has been added to the features we're polyfilling from the polyfill service we're using. `findIndex` might come with the default set, but should be double checked: https://github.com/zooniverse/Panoptes-Front-End/blob/master/views/index.ejs#L74

Or `value.indexOf()` could be used like @eatyourgreens is suggesting.